src/geometry/circle-circle-intersection.md

```diff
@@ -32,3 +32,5 @@ The only degenerate case we need to consider separately is when the centers of t
 ## Practice Problems
 
 [RadarFinder](https://community.topcoder.com/stat?c=problem_statement&pm=7766)
+[Runaway to a shadow - Codeforces](http://codeforces.com/problemset/problem/681/E)
```

Comment:
Hey,
Sorry about that. I corrected that and have also added bullets to the problems, to make it clearer. I hope you'll like the idea.